aeron-test-support/src/main/java/io/aeron/test/cluster/TestClusterClock.java

```diff
@@ -50,4 +50,15 @@ public void update(final long tick, final TimeUnit tickTimeUnit)
     {
         this.tick = tickTimeUnit.convert(tick, tickTimeUnit);
     }
+
+    public synchronized long increment(final long tick, final TimeUnit tickTimeUnit)
+    {
+        this.tick += tickTimeUnit.convert(tick, tickTimeUnit);
+        return this.tick;
+    }
+
+    public synchronized long increment(final long tick)
```

Comment:
The `synchronized` can be dropped here since it delegates to the second `increment` and the `timeUnit` is `final`.

Reply:
I did that to stop Intellij moaning at me about the use of an increment on the volatile field.  Probably best practice to make sure it is correct, but it should really be an atomic.  I'll fix that.